doc/modules/model_evaluation.rst

```diff
@@ -102,11 +102,27 @@ Scoring                                Function
 'neg_mean_poisson_deviance'            :func:`metrics.mean_poisson_deviance`
 'neg_mean_gamma_deviance'              :func:`metrics.mean_gamma_deviance`
 'neg_mean_absolute_percentage_error'   :func:`metrics.mean_absolute_percentage_error`
-'d2_absolute_error_score'              :func:`metrics.d2_absolute_error_score`
-'d2_pinball_score'                     :func:`metrics.d2_pinball_score`
-'d2_tweedie_score'                     :func:`metrics.d2_tweedie_score`
 ====================================   ==============================================     ==================================
 
+The following metrics functions are not implemented as named
+scorers. They cannot be passed to the ``scoring`` parameters; instead
+their callable needs to be passed to :func:`make_scorer` together with the
+value of the user-settable parameters. See :ref:`scoring` for more
+details on :func:`make_scorer` usage.
+
+=====================================  =========  ==============================================
```

Comment:
I think something is wrong with the syntax since the table disappeared on the HTML rendering:

![image](https://github.com/scikit-learn/scikit-learn/assets/7454015/3fb33406-44c2-40b8-ad26-4b2e06eddf8b)